milabench/config.py

```diff
@@ -1,10 +1,16 @@
+import io
 import yaml
 
 from .fs import XPath
 from .merge import self_merge
 
 
 def parse_config(config_file, base=None):
+    benchmark = None
+    # It might be a yaml
+    if not XPath(config_file).exists():
+        benchmark = yaml.safe_load(io.StringIO(config_file))
+        config_file = benchmark["config_file"]
```

Comment:
Not sure what the purpose of this is.

Reply:
only useful when in `milabench dev` as you don't need to specify again the select but not necessary